lib/runtime/wasmer/imports.go

```diff
@@ -1580,9 +1591,9 @@ func storageAppend(storage runtime.Storage, key, valueToAppend []byte) error {
 	}
 
 	// append new length prefix to start of items array
-	finalVal := append(lengthEnc, valueRes...)
-	logger.Debug("[ext_storage_append_version_1]", "resulting value", fmt.Sprintf("0x%x", finalVal))
-	storage.Set(key, finalVal)
+	lengthEnc = append(lengthEnc, valueRes...)
+	logger.Debug("[ext_storage_append_version_1]", "resulting value", fmt.Sprintf("0x%x", lengthEnc))
+	storage.Set(key, lengthEnc)
```

Comment:
why is this updated? I think `finalVal` is more descriptive, as this isn't an encoded length

Reply:
This was updated because DeepSource was complaining that append was appending to a variable with a different name.  Yes, finalVal is more descriptive, I've update to use that instead.